mongodbatlas/search.go

```diff
@@ -280,7 +280,7 @@ type IndexField struct {
 	IndexShapes    *bool                  `json:"indexShapes,omitempty"`
 }
 
-// SearchAnalyzer custom analyzer definition.
+// SearchAnalyzer search analyzer definition.
```

Comment:
if updating this be mindful to write it like simple sentence so this page can be read
https://pkg.go.dev/go.mongodb.org/atlas@v0.8.0/mongodbatlas#SearchAnalyzer